library/data/tags/src/main/java/org/quiltmc/qsl/tag/api/package-info.java

```diff
@@ -6,24 +6,25 @@
  * Tags are a way to have un-hardcoded list of registry entries since the content of tags are controlled by data packs.
  * For example, Minecraft use those to identify all fence blocks in the game, which is used for the connection mechanic.
  * <p>
- * By default tags are available for most of the static registries and are one of the earliest resources to be loaded on
- * server start.
+ * By default tags are available for every registries of the game, they are entirely controlled by the server.
  * <p>
- * The goal of the API is to give to modders the ability to get (and register) tags of any registries, dynamic ones included.
+ * The goal of this API is to give to modders the ability to iterate through tags, have client-only tags,
+ * and client fallbacks for server-controlled tags.
  *
  * <p>
  * <h3>Tag Types</h3>
  * {@link org.quiltmc.qsl.tag.api.TagType} allows categorizing tags and defining extra behavior.
- * Those types allow defining if a tag is required for a server to start, for a client to connect,
- * if a tag is only loaded on the client, etc.
+ * Those types allow defining if a tag has a client fallback, if a tag is only loaded on the client, etc.
  *
  * <p>
- * <h3>Get and register a tag</h3>
- * To get and/or register a tag, use {@link org.quiltmc.qsl.tag.api.TagRegistry}'s create methods.
- * All registered tags through this interface will auto-update their content through data/resource pack reloads.
- * You can find built-in tag registries for some of the most relevant registries, you can also create a new tag registry
- * using {@link org.quiltmc.qsl.tag.api.TagRegistry#of(net.minecraft.util.registry.RegistryKey, java.lang.String)}
- * or {@link org.quiltmc.qsl.tag.api.TagRegistry#of(java.util.function.Supplier)} if needed.
+ * <h3>Use the new tag types</h3>
+ * To use one of the new tag types, use {@link org.quiltmc.qsl.tag.api.QuiltTagKey#create(net.minecraft.util.registry.RegistryKey, net.minecraft.util.Identifier, org.quiltmc.qsl.tag.api.TagType)}.
+ * It allows to create a new {@link net.minecraft.tag.TagKey} but with a specific {@link org.quiltmc.qsl.tag.api.TagType} instead of the default one.
```

Comment:
```suggestion
 * It allows creating a new {@link net.minecraft.tag.TagKey} but with a specific {@link org.quiltmc.qsl.tag.api.TagType} instead of the default one.
```